### PR TITLE
cam6_4_003:  replace L93 ncdata files w/ corrected versions

### LINKED ISSUE #1034: Problematic hybrid coefficients in L93
### What happened?

I discovered some ill conditioned aspects of the L93 grid when trying to run ne480pg3, which supports higher mountain tops and smaller surface pressures than coarser grids. @JulioTBacmeister has generated profiles of the level thickness for different values of the surface pressure, and as you can see for a 400 hPa surface pressure, level 62 goes negative:

Screenshot 2024-05-17 at 11 25 03 AM

The problem seems to be some type of kink in the coef's between 300 and 100 hPa (black line):
Screenshot 2024-05-17 at 11 25 19 AM

The red line corrects for this kink, and Julio has performed science validation for this new grid. 

The problematic grid is from this ncdata file in ```namelist_defaults.xml```
```
<ncdata dyn="se" hgrid="ne30np4" npg="3" nlev="93">atm/cam/inic/se/cam7_FMT_ne30pg3_mg17_L93_c221118.nc</ncdata>
```

HOWEVER, we have not checked the other L93 ncdata files in ```namelist_defaults.xml```
```
<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.cam6_3_112.FCMTHIST_v0c.ne30.non-ogw-ubcT-effgw0.7.001.cam.i.1998-01-01-00000_c230810.nc</ncdata>
<ncdata dyn="se" hgrid="ne3np4"   nlev="93"             ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_topo_ne3pg3_mg37_L93_01-01-31_c221214.nc</ncdata>
<ncdata dyn="se" hgrid="ne3np4"   nlev="93"  aquaplanet="1" ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_aqua_ne3pg3_mg37_L93_01-01-31_c221214.nc</ncdata>
```

I'm going to look into those and post an update to this thread.

### What are the steps to reproduce the bug?

Run any L93 configuration.

### What CAM tag were you using?

cam6_3_153

### What machine were you running CAM on?

CISL machine (e.g. cheyenne)

### What compiler were you using?

Intel

### Path to a case directory, if applicable

/glade/derecho/scratch/aherring/cam6_3_153_FMTHIST_ne480pg3_ne480pg3_mt13_28800pes_240430_nogw_test.005

### Will you be addressing this bug yourself?

Yes

### Extra info

_No response_

### PR DESCRIPTION
**Pull request by adamrher**
### **What happened?**
 L93 hybrid coefficients had a discontinuous kink, or offset, creating an anomalously thin layer in the 300 hPa - 100 hPa altitude range. This region overlaps with the L58 grid, but for some reason the L58 grid didn't get contaminated like it was in L93. We aren't sure how this happened.
More here: https://github.com/ESCOMP/CAM/issues/1034 

@jtruesdal I'm requesting your review because you had introduced these files in https://github.com/ESCOMP/CAM/pull/815, and I would like your advice on how to rename these files.

### CODE REVIEW

**Review by cacraigucar — CHANGES_REQUESTED** [review]
Just a reminder, that the mandatory metadata needs to be added to these files.  See: https://www.cesm.ucar.edu/models/cam/metadata

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 1]
```diff
@@ -255,7 +255,7 @@
 
 <ncdata dyn="se" hgrid="ne3np4"   nlev="32"  aquaplanet="1" ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_aqua_ne3pg3_mg37_L32_01-01-31_c221214.nc</ncdata>
 <ncdata dyn="se" hgrid="ne3np4"   nlev="58"  aquaplanet="1" ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_aqua_ne3pg3_mg37_L58_01-01-31_c221214.nc</ncdata>
-<ncdata dyn="se" hgrid="ne3np4"   nlev="93"  aquaplanet="1" ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_aqua_ne3pg3_mg37_L93_01-01-31_c221214.nc</ncdata>
+<ncdata dyn="se" hgrid="ne3np4"   nlev="93"  aquaplanet="1" ic_ymd="101" >/glade/derecho/scratch/aherring/restart/L93v2/cam6_3_153_QPC6_ne3pg3_ne3pg3_mg37_36pes_240518_L93v2.001.cam.i.0001-02-01-00000.nc</ncdata>
```
Comment (jtruesdal) [inline bld/namelist_files/namelist_defaults_cam.xml:258]:
```suggestion
<ncdata dyn="se" hgrid="ne3np4"   nlev="93"  aquaplanet="1" ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_aqua_ne3pg3_mg37_L93_01_02_01_c240518.nc</ncdata>
```

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 2]
```diff
@@ -220,7 +220,7 @@
 
 <ncdata dyn="se" hgrid="ne3np4"   nlev="32"             ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_topo_ne3pg3_mg37_L32_01-01-31_c221214.nc</ncdata>
 <ncdata dyn="se" hgrid="ne3np4"   nlev="58"             ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_topo_ne3pg3_mg37_L58_01-01-31_c221214.nc</ncdata>
-<ncdata dyn="se" hgrid="ne3np4"   nlev="93"             ic_ymd="101" >atm/cam/inic/se/cam6_QPC6_topo_ne3pg3_mg37_L93_01-01-31_c221214.nc</ncdata>
+<ncdata dyn="se" hgrid="ne3np4"   nlev="93"             ic_ymd="101" >/glade/derecho/scratch/aherring/restart/L93v2/cam6_3_153_FMTHIST_ne3pg3_ne3pg3_mg37_36pes_240517_L93v2.001.cam.i.1979-02-01-00000.nc</ncdata>
```
Comment (jtruesdal) [inline bld/namelist_files/namelist_defaults_cam.xml:223]:
```suggestion
<ncdata dyn="se" hgrid="ne3np4"   nlev="93"             ic_ymd="101" >atm/cam/inic/se/cam6_FMTHIST_ne3pg3_mg37_L93_79-02-01_c240517.nc</ncdata>
```

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 3]
```diff
@@ -125,10 +125,10 @@
...
 <ncdata dyn="se" hgrid="ne0np4CONUS.ne30x8" nlev="32" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.e22.FCnudged.ne0CONUSne30x8_ne0CONUSne30x8_mt12.cam6_2_032.002.cam.i.2013-01-01-00000_c200623.nc</ncdata>
 <ncdata dyn="se" hgrid="ne30np4" nlev="58" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.e22.FCnudged.ne30_ne30_mg17.release-cesm2.2.0_spinup.2010_2020.001.cam.i.2011-01-01-00000_L58_c220310.nc</ncdata>
-<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.cam6_3_112.FCMTHIST_v0c.ne30.non-ogw-ubcT-effgw0.7.001.cam.i.1998-01-01-00000_c230810.nc</ncdata>
+<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">/glade/derecho/scratch/aherring/restart/L93v2/f.cam6_3_160.FCMT_ne30.moving_mtn.001.cam.i.1996-01-01-00000.nc</ncdata>
 
 <ncdata dyn="se" hgrid="ne30np4" npg="3" nlev="58">atm/cam/inic/se/FLT_L58_ne30pg3_IC_c220623.nc</ncdata>
-<ncdata dyn="se" hgrid="ne30np4" npg="3" nlev="93">atm/cam/inic/se/cam7_FMT_ne30pg3_mg17_L93_c221118.nc</ncdata>
+<ncdata dyn="se" hgrid="ne30np4" npg="3" nlev="93">/glade/derecho/scratch/aherring/restart/L93v2/c153_ne30pg3_FMTHIST_x02.cam.i.1990-01-01-00000.nc</ncdata>
```
Comment (jtruesdal) [inline bld/namelist_files/namelist_defaults_cam.xml:131]:
```suggestion
<ncdata dyn="se" hgrid="ne30np4" npg="3" nlev="93">atm/cam/inic/se/c153_ne30pg3_FMTHIST_x02.cam.i.1990-01-01-00000_c240618.nc</ncdata>
```

File: bld/namelist_files/namelist_defaults_cam.xml [hunk 4]
```diff
@@ -125,10 +125,10 @@
 <ncdata dyn="se" hgrid="ne0np4CONUS.ne30x8" nlev="32" chem="trop_strat_mam4_vbs">atm/cam/inic/se/f.e22.FCnudged.ne0CONUSne30x8_ne0CONUSne30x8_mt12.cam6_2_032.002.cam.i.2013-01-01-00000_c200623.nc</ncdata>
 <ncdata dyn="se" hgrid="ne0np4CONUS.ne30x8" nlev="32" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.e22.FCnudged.ne0CONUSne30x8_ne0CONUSne30x8_mt12.cam6_2_032.002.cam.i.2013-01-01-00000_c200623.nc</ncdata>
 <ncdata dyn="se" hgrid="ne30np4" nlev="58" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.e22.FCnudged.ne30_ne30_mg17.release-cesm2.2.0_spinup.2010_2020.001.cam.i.2011-01-01-00000_L58_c220310.nc</ncdata>
-<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.cam6_3_112.FCMTHIST_v0c.ne30.non-ogw-ubcT-effgw0.7.001.cam.i.1998-01-01-00000_c230810.nc</ncdata>
+<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">/glade/derecho/scratch/aherring/restart/L93v2/f.cam6_3_160.FCMT_ne30.moving_mtn.001.cam.i.1996-01-01-00000.nc</ncdata>
```
Comment (jtruesdal) [inline bld/namelist_files/namelist_defaults_cam.xml:128]:
```suggestion
<ncdata dyn="se" hgrid="ne30np4" nlev="93" chem="trop_strat_mam5_vbs">atm/cam/inic/se/f.cam6_3_160.FCMT_ne30.moving_mtn.001.cam.i.1996-01-01-00000_c240618.nc</ncdata>
```

**Review by jtruesdal — CHANGES_REQUESTED** [review]
@adamrher I've added the metadata and renamed the files.  Here are the recommended changes.  I'll push these to the data repo once the PR is approved.

File: bld/namelist_files/namelist_defaults_cam.xml (hunk 4 above)
Comment (adamrher) [reply bld/namelist_files/namelist_defaults_cam.xml:128]: Thanks John.

**Comment by cacraigucar** [comment]
Just checking to see which of you is going to make the changes the @jtruesdal provided.  Once that is done, I can complete my review and schedule this for CAM tagging.

**Comment by jtruesdal** [comment]
@adamrher @cacraigucar I'll make the changes.  Just checking with Adam to see if I have the correct permissions to merge.

**Review by cacraigucar — APPROVED** [review]

**Review by jtruesdal — APPROVED** [review]

**Pull request merged**